cupyx/scipy/ndimage/_filters_generic.py

```diff
@@ -0,0 +1,538 @@
+import cupy
+
+
+from cupyx.scipy.ndimage._filters_core import _generate_nd_kernel
```

Comment:
```suggestion
from cupyx.scipy.ndimage import _filters_core
```


based on the [CuPy style guide](https://docs.cupy.dev/en/v4.5.0/contribution.html#coding-guidelines)
```
    Relative imports are prohibited. [H304]
    Importing non-module symbols is prohibited.
    Import statements must be organized into three parts: standard libraries, third-party libraries, and internal imports. [H306]
```